src/sentry/integrations/vsts/integration.py

```diff
@@ -266,7 +296,12 @@ class VstsIntegrationProvider(IntegrationProvider):
     oauth_redirect_url = '/extensions/vsts/setup/'
     needs_default_identity = True
     integration_cls = VstsIntegration
-    features = frozenset([IntegrationFeatures.ISSUE_SYNC, IntegrationFeatures.COMMITS])
+
+    features = frozenset([
+        IntegrationFeatures.ISSUE_SYNC,
```

Comment:
why is `IntegrationFeatures.ISSUE_SYNC` in here twice? Should it be `ISSUE_BASIC` as well?

Reply:
I think you commented on an old revision

Reply:
i blame github